common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -120,25 +142,75 @@ public void render(PoseStack poseStack, float partialTicks, Window window) {
                     healthTexture.getTextureY1(),
                     81,
                     healthTexture.getTextureY2(),
-                    (flip ? -ActionBarModel.getCurrentHealth() : ActionBarModel.getCurrentHealth())
-                            / (float) ActionBarModel.getMaxHealth());
+                    progress);
+        }
+
+        protected void renderText(PoseStack poseStack, float renderY, String text) {
+            FontRenderer.getInstance()
+                    .renderAlignedTextInBox(
+                            poseStack,
+                            text,
+                            this.getRenderX(),
+                            this.getRenderX() + this.getWidth(),
+                            renderY,
+                            0,
+                            this.textColor,
+                            this.getRenderHorizontalAlignment(),
+                            this.textShadow);
+        }
+    }
+
+    public static class BloodPoolBarOverlay extends HealthBarOverlay {
+        public BloodPoolBarOverlay() {
+            super(
+                    new OverlayPosition(
+                            -30,
+                            -150,
+                            VerticalAlignment.Bottom,
+                            HorizontalAlignment.Center,
+                            OverlayPosition.AnchorSection.BottomMiddle),
+                    new GuiScaledOverlaySize(81, 21));
         }
 
         @Override
-        protected void onConfigUpdate(ConfigHolder configHolder) {}
+        public void render(PoseStack poseStack, float partialTicks, Window window) {
+            if (!WynnUtils.onWorld()) return;
+
+            LerpingBossEvent poolEvent = null;
+            String bloodPoolPercent = "";
```

Comment:
This is not a rendering thing. Can you extract this into some Util class (one existing, or invent a new if none seem good enough), so other code in the future that wants to read the blood pool value can do so easily? (Same for the Mana Bank)

Reply:
@kristofbolyai Did you miss this, or intentionally skip doing it?

Reply:
ooh, I forgot. I will do this later separately.

Reply:
I will keep tabs on this and do it.

Reply:
Thanks! <3